org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/preferences/BuilderPreferenceAccess.java

```diff
@@ -32,6 +33,37 @@
 	 * Name of a preference for configuring whether the builder participant is enabled or not.
 	 */
 	public static final String PREF_AUTO_BUILDING = "autobuilding"; //$NON-NLS-1$
+	/**
+	 * Name of a preference defining the scheduling rule strategy for the builder.
+	 * @since 2.16
```

Comment:
Nitpicking here, shouldn't the description and `@since` be separated by a newline?

Reply:
correct, we usually do this for methods